cfgrammar/src/lib/mod.rs

```diff
@@ -53,7 +53,7 @@
 use serde::{Deserialize, Serialize};
 
 mod idxnewtype;
-pub mod utils;
+pub mod span;
```

Comment:
On reflection, we should also `use span::Span` so most people can just do `use cfgrammar::Span` without worrying about the module it's defined in. [This also makes the `crate::span::Span` dance simpler as just `crate::Span`.]